src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -193,7 +197,12 @@ export class ExposureNotificationService {
   }
 
   private async init() {
-    const exposureStatus = JSON.parse((await this.storage.getItem(EXPOSURE_STATUS)) || 'null');
+    let exposureStatus;
+    try {
+      exposureStatus = JSON.parse((await this.secureStorage.get(EXPOSURE_STATUS)) || 'null');
+    } catch (error) {
+      exposureStatus = 'null';
```

Comment:
This should be `null` value, not null string.

Reply:
Let's do this without try/catch

```ts
exposureStatus = JSON.parse((await this.secureStorage.get(EXPOSURE_STATUS).catch(() => null)) || 'null');
```

Reply:
> This should be `null` value, not null string.

I was wondering about that, but in the event that line 202 is unable to parse the JSON, it sets it to the same 'null' string.

Reply:
`JSON.parse('null')` will be `null` value (not string).